client/src/Movies/Movie.js

```diff
@@ -26,43 +26,23 @@ export default class Movie extends Component {
       });
   };
   // Uncomment this code when you're ready for the stretch problems
-  // componentWillReceiveProps(newProps){
-  //   if(this.props.match.params.id !== newProps.match.params.id){
-  //     this.fetchMovie(newProps.match.params.id);
-  //   }
-  // }
+  componentWillReceiveProps(newProps){
```

Comment:
This will get deprecated in React 17, it'll be replace with `getDerivedStateFromProps`